Fix PretrainedBERT init

hepaajan opened the pull request:
new_weights was renamed to init_weights_and_state and weights are stored
in self.weights

afrozenator commented:
Hi @hepaajan -- Can you sign the CLA as instructed by the bot?

hepaajan commented:
Oh, sorry, This slipped my mind,  now it's signed. Let's wait for the bot to pick it up :)

hepaajan commented:
Rebased and fixed my email (which was accidentally work email)

afrozenator commented:
Thanks! You'll still need to reply with `@googlebot I signed it!` for it to
check again.

On Mon, Nov 16, 2020 at 11:38 PM hepaajan <notifications@github.com> wrote:

> Rebased and fixed my email (which was accidentally work email)
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/google/trax/pull/1135#issuecomment-728745892>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAIUEFSYIFDT5P4Z7ITIIH3SQISANANCNFSM4SZKG5MQ>
> .
>


-- 
We are here on earth to do good for others. What the others are here for, I
don't know.

Afroz Mohiuddin

hepaajan commented:
It seems it automatically checked it again. It should ok now as googlebot added a "cla:yes" label
> Thanks! You'll still need to reply with `@googlebot I signed it!` for it to check again.
> […](#)
> On Mon, Nov 16, 2020 at 11:38 PM hepaajan ***@***.***> wrote: Rebased and fixed my email (which was accidentally work email) — You are receiving this because you commented. Reply to this email directly, view it on GitHub <[#1135 (comment)](https://github.com/google/trax/pull/1135#issuecomment-728745892)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AAIUEFSYIFDT5P4Z7ITIIH3SQISANANCNFSM4SZKG5MQ> .
> -- We are here on earth to do good for others. What the others are here for, I don't know. Afroz Mohiuddin

afrozenator commented:
Ah, nice! Didn't see that, sorry.

On Tue, Nov 17, 2020, 8:40 AM hepaajan <notifications@github.com> wrote:

> It seems it automatically checked it again. It should ok now as googlebot
> added a "cla:yes" label
>
> Thanks! You'll still need to reply with @googlebot I signed it! for it to
> check again.
> … <#m_6933270259695040512_>
> On Mon, Nov 16, 2020 at 11:38 PM hepaajan *@*.***> wrote: Rebased and
> fixed my email (which was accidentally work email) — You are receiving this
> because you commented. Reply to this email directly, view it on GitHub <#1135
> (comment)
> <https://github.com/google/trax/pull/1135#issuecomment-728745892>>, or
> unsubscribe
> https://github.com/notifications/unsubscribe-auth/AAIUEFSYIFDT5P4Z7ITIIH3SQISANANCNFSM4SZKG5MQ
> .
> -- We are here on earth to do good for others. What the others are here
> for, I don't know. Afroz Mohiuddin
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/google/trax/pull/1135#issuecomment-729050919>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAIUEFUAZBQN42PB5CBMIYDSQKROTANCNFSM4SZKG5MQ>
> .
>
